### DISCUSSION TITLE
States missing in the docs page

### DISCUSSION THREAD

**Discussion by TheZoker** [raomin/ESPAltherma#374] [Q&A] [answered]
On the docs page:
https://raomin.github.io/ESPAltherma/

The states are missing in the home assistant sections:
![image](https://github.com/raomin/ESPAltherma/assets/1368405/595e6302-e6ef-43ce-942a-3f89f1a924ab)

On the main README of this repo they are there:
![image](https://github.com/raomin/ESPAltherma/assets/1368405/45cff701-a2c5-4394-bdf0-d49d700cd200)

To avoid future confusion it would be great, if these could also be added to the docs page. Most people may just copy-paste this code. Thanks!

**Comment by raomin** [comment, marked as answer]
Good point. It's actually the same page but `{{ }}` seems to be interpreted in the web version.
I unpublished it anyway (but it's still online though...)
